packages/nuxt/src/app/components/nuxt-stubs.ts

```diff
@@ -0,0 +1,17 @@
+import { createError } from '../composables/error'
+
+function renderStubMessage (name: string) {
+  throw createError({
+    fatal: true,
+    statusCode: 500,
```

Comment:
Using fatal flag, can cause to have unwanted side-effects like showing verbose Nitro logs and decreasing DX. Also we could use something better than 500 for this like 404 even as we use for placeholder assets too

Reply:
It is necessary to make the error show on client-side. (as opposed to SSR)

Reply:
Yeah. It could be better with placeholder rendering instead of failing whole page but we can certainly iterate to make it happen :)

Reply:
Initially I did show a placeholder but didn't have time to complete design. Switched to full error page until design is complete 😁